### PR TITLE
Retry specifics guidance/entity framework core

### PR DESCRIPTION
**Pull request by luisgmsft**

### CODE REVIEW

**Comment by bennage** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/mspnp/architecture-center, working directory: W:\5dog-s, target branch: master, target branch commit: 1db6863c030785d5d5f1c3f33fe34ba1899c5edf, source branch: 9c4f9dbcf81ea52d79d0d7e43ae5f930/retry-specifics-guidance/entity-framework-core, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C8%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705082053033541-96%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/mspnp/architecture-center, working directory: W:\ou1c-s, target branch: master, target branch commit: 1db6863c030785d5d5f1c3f33fe34ba1899c5edf, source branch: 9c4f9dbcf81ea52d79d0d7e43ae5f930/retry-specifics-guidance/entity-framework-core, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C8%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705082101439538-96%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/mspnp/architecture-center, working directory: W:\vwgu-s, target branch: master, target branch commit: 1db6863c030785d5d5f1c3f33fe34ba1899c5edf, source branch: 9c4f9dbcf81ea52d79d0d7e43ae5f930/retry-specifics-guidance/entity-framework-core, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C8%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705082346390747-96%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/best-practices/retry-service-specific.md [hunk 1]
```diff
@@ -362,7 +362,7 @@ Retry support is provided when accessing SQL Database using Entity Framework Cor
 * If the specified retry count is exceeded, the results are wrapped in a new exception. It does not bubble up the current exception.
 
 ### Examples
-The following snippets, are extracted from the [Connection Resiliency](/ef/core/miscellaneous/connection-resiliency) document, which describes different scopes and scenarios on retry for SQL Server and other custom scenarios.
+The following snippets, are extracted from the [Connection Resiliency](https://docs.microsoft.com/en-us/ef/core/miscellaneous/connection-resiliency) document, which describes different scopes and scenarios on retry for SQL Server and other custom scenarios.
```
Comment (bennage) [inline docs/best-practices/retry-service-specific.md]: This was actually correct. We use relative links to content hosted on docs.microsoft.com.
We should also avoid including the locale.

**Comment by bennage** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/mspnp/architecture-center, working directory: W:\3gpo-s, target branch: master, target branch commit: 1db6863c030785d5d5f1c3f33fe34ba1899c5edf, source branch: 9c4f9dbcf81ea52d79d0d7e43ae5f930/retry-specifics-guidance/entity-framework-core, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C8%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705082356269757-96%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by luisgmsft** [comment]
Merge fails continuously for reasons I can't understand. I'll recreate it from scratch and try again.

**Pull request closed without merging**